src/ai/backend/manager/models/git_token.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+import datetime
+import json
+import logging
+import uuid
+from typing import Sequence
+
+import graphene
+import sqlalchemy as sa
+from graphene.types.datetime import DateTime as GQLDateTime
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.ext.asyncio import AsyncConnection as SAConnection
+from sqlalchemy.sql import text
+
+from ai.backend.common.logging import BraceStyleAdapter
+
+from .base import GUID, Base, metadata
+
+log = BraceStyleAdapter(logging.getLogger(__spec__.name))  # type: ignore[name-defined]
+
+
+__all__: Sequence[str] = (
+    "git_tokens",
+    "GitToken",
+    "GitTokenRow",
+    "insert_update_git_tokens",
+)
+
+git_tokens = sa.Table(
+    "git_tokens",
+    metadata,
+    sa.Column(
+        "user_id",
+        GUID,
+        sa.ForeignKey("users.uuid", onupdate="CASCADE", ondelete="CASCADE"),
+        index=True,
+        nullable=False,
+        primary_key=True,
+    ),
+    sa.Column("domain", sa.String(length=200), unique=True, primary_key=True, nullable=False),
```

Comment:
And `domains` table has `name` field which is a primary key.
How about defining this field as a `ForeignKey`?